Podfile

```diff
@@ -12,4 +12,6 @@ target 'Decred Wallet' do
   pod 'Fabric'
   pod 'Crashlytics'
 
+  # Signals Library for reactiveness
+  pod 'Signals', '~> 6.0'
```

Comment:
I haven't seen the use of `Signals` so far in this PR. Please remove if not used. If it is indeed used, bring it up for discussion in the dev room.